help_data.yaml

```diff
@@ -30,12 +30,13 @@ esm-apps:
 
 esm-infra:
    help: |
-     UA Infra: Extended Security Maintenance is enabled by default on entitled
```

Comment:
This modification will break two integration tests for the help command:

https://github.com/canonical/ubuntu-advantage-client/blob/master/features/attached_commands.feature#L300
https://github.com/canonical/ubuntu-advantage-client/blob/master/features/unattached_commands.feature#L89